packages/sources/cryptoid/src/config.ts

```diff
@@ -1,6 +1,8 @@
 import { Requester } from '@chainlink/ea-bootstrap'
 import { Config } from '@chainlink/types'
 
+export const NAME = 'CRYPTO-ID'
```

Comment:
We should keep dashes out of `NAME` as they are not supported in environment variable names
https://pubs.opengroup.org/onlinepubs/009695399/basedefs/xbd_chap08.html

```suggestion
export const NAME = 'CRYPTO_ID'
```